DigitalLearningSolutions.Web/Views/Supervisor/SignOffProfileAssessment.cshtml

```diff
@@ -201,7 +204,18 @@
             <span asp-validation-for="SignedOff" class="text-danger"></span>
           </div>
         </nhs-form-group>
-      </fieldset>
+                @if (Model.NumberOfSelfAssessedOptionalCompetencies >= Model.SelfAssessmentResultSummary.MinimumOptionalCompetencies)

```

Comment:
As discussed over a call, let's keep the original scope of the ticket and avoid this unnecessary condition.

Reply:
Done